static/docs/commands-reference/commit.md

```diff
@@ -211,7 +213,7 @@ execute this set of commands:
 ```dvc
 $ dvc commit
 $ dvc status
-Pipeline is up to date. Nothing to reproduce.
+Pipeline(s) is (are) up to date. Nothing to reproduce.
```

Comment:
the same concern as in the DVC core repo. It does not look nice to me.

Reply:
I was waiting for https://github.com/iterative/dvc/pull/2281 to merge )

Updated in 3dc5489